inst/include/cpp11/nameof.hpp

```diff
@@ -0,0 +1,82 @@
+/// Distributed under the Boost Software License, Version 1.0. (See
+/// http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
Is this copied from a boost library ? Should this be acknowledged in the `DESCRIPTION` file ?